src/coreclr/jit/compiler.h

```diff
@@ -8438,6 +8438,8 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
                                                       CorInfoType simdBaseJitType,
                                                       bool        isSimdAsHWIntrinsic = false)
     {
+        assert(varTypeIsSIMD(simdType));
```

Comment:
Should we likewise add an assert to `gtGetStructHandleForSIMD` and `gtGetStructHandleForHWSIMD`?

Reply:
Good idea, added